libs/pika/runtime/src/runtime.cpp

```diff
@@ -232,30 +232,31 @@ namespace pika {
     ///////////////////////////////////////////////////////////////////////////
     namespace strings {
         char const* const runtime_state_names[] = {
-            "state_invalid",         // -1
-            "state_initialized",     // 0
-            "state_pre_startup",     // 1
-            "state_startup",         // 2
-            "state_pre_main",        // 3
-            "state_starting",        // 4
-            "state_running",         // 5
-            "state_suspended",       // 6
-            "state_pre_sleep",       // 7
-            "state_sleeping",        // 8
-            "state_pre_shutdown",    // 9
-            "state_shutdown",        // 10
-            "state_stopping",        // 11
-            "state_terminating",     // 12
-            "state_stopped"          // 13
+            "state::invalid",         // -1
```

Comment:
We may want to consider renaming it `runtime_state` instead. Not for this PR, just something to think about...

Reply:
And for that matter we might want to use separate enums for the runtime and scheduler states.

https://github.com/pika-org/pika/issues/447